CMakeLists.txt

```diff
@@ -85,12 +85,17 @@ configure_file("${CMAKE_SOURCE_DIR}/project_version.h.in" "${CMAKE_BINARY_DIR}/p
 
 # Code checks
 include("CodeStyle")
-include("Test")
+include("Cpp")
 include("Lint")
+include("Test")
 include("FindVulkanVersion")
 
 # Apply misc build directives to the given target
 macro(common_build_directives TARGET)
+    if(MSVC)
+        target_compile_options(${TARGET} PRIVATE $<$<CONFIG:Debug>:/MTd> $<$<CONFIG:Release>:/MT>)
```

Comment:
This appears to be generating linker warnings with Release builds and errors with Debug builds.  If you remove the target_compile_options addition, I think this should be good to merge.

Reply:
I'll try to repro this, but I don't remember seeing this on the amd-builds.

Reply:
OK, found the issue.
For some reason I can only repro on my test system - I suspect it's related to what is installed on the system.
The issue is that LZ4 static lib is linked against the dynamic runtime (/MD).

Reply:
Removed the /MT and /MTd options when compiling with MSVC